docs/core_concepts/15_authentification/index.mdx

```diff
@@ -0,0 +1,101 @@
+import DocCard from '@site/src/components/DocCard';
+
+# Authentification
+
+Windmill provides flexible authentication options to ensure secure access to the platform. Users can authenticate through Single Sign-On (SSO) with a restricted domain or with a simple e-mail if added manually by superadmins.
+
+<div class="grid grid-cols-2 gap-6 mb-4">
+	<DocCard
+    	title="Roles and Permissions"
+    	description="Find out about the roles within a Windmill instance and their respective permissions."
+    	href="/docs/core_concepts/roles_and_permissions"
+    />
+</div>
+
+## Restricted Domain Authentication
+
+Windmill supports authentication [through SSO](../../misc/2_setup_oauth/index.md) for users with email addresses from a restricted domain. This allows organizations to control access to Windmill based on their domain policy. Users with email addresses from the authorized domain can authenticate seamlessly using their SSO credentials.
+
+To enable restricted domain authentication, an administrator can configure the authorized domain in Windmill's settings. Once configured, users with email addresses from the authorized domain will be able to log in using their SSO provider.
+
+## Manually Add Users to a Windmill Instance
+
+As a superadmin of the workspace, you have the ability to manually add users to the Windmill instance. This is useful for inviting users who do not have SSO credentials or for providing access to individuals outside the restricted domain.
+
+To manually add users:
+1. Log in to the Windmill instance as a superadmin.
+2. Click on your username and pick "Superadmin settings".
+3. Fill:
+   - Email: The email address of the user.
+   - Password: A password for the user's account.
+   - Name (Optional): The name of the user.
+   - Company (Optional): The company or organization the user belongs to.
+4. "Add user to instance".
+
+![Manually Add Users](./add_global_users.png "Manually Add Users")
+
+If SMTP is configured, an email will be sent to the user with their account details and instructions for accessing Windmill.
+
+By default, users are not invited to any workspace, unless auto-invite has been set-up.
+
+## Adding Users to a Workspace
+
+Once added to an instance, users can create their own workspace (on cloud, those ones are not considered as Premium as plans work at the workspace-level). However, by default they will not be invited to any workspace.
```

Comment:
> (on cloud, those ones are not considered as Premium as plans work at the workspace-level)

remove or clarify, I would remove

Also add that windmill can be configured to remove the ability for non-superadmins to create workspace using the: CREATE_WORKSPACE_REQUIRE_SUPERADMIN env variable.